ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -32,14 +32,16 @@ bundle exec rubocop -D
 
 ## Brakeman
 
-```ruby
-group :development, :test do
-  gem 'brakeman', require: false
-end
+Brakeman comes byb default with Rails. Add it to the `bin/fastcheck` script.
+
+```
+bundle exec brakeman -q -z --no-summary --no-pager
 ```
 
 ## Mdl
 
+An optional check for markdown files. You can include it or not. Discuss within your team.
```

Comment:
Maybe add example usage of mdl?
```
bundle exec mdl $(find . -name '*.md'|grep -E -v '(SUMMARY.md|vendor/)')
```

Reply:
The template includes an example. I don't want to duplicate it here